*** ./smali/com/android/server/ServerThread.smali	2013-10-21 03:27:14.675587487 +0800
--- /home/xiangxin/patchrom/MIUIV5/st25i/temp/new_smali/services.jar.out/./smali/com/android/server/ServerThread.smali	2013-10-21 03:29:03.323586896 +0800
***************
*** 113,118 ****
--- 113,121 ----
  
  .method public run()V
      .locals 120
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->CHANGE_CODE:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
  
      .prologue
      const/16 v3, 0xbc2
***************
*** 536,546 ****
  
      invoke-static {v3, v9}, Landroid/util/Slog;->i(Ljava/lang/String;Ljava/lang/String;)I
  
!     new-instance v75, Lcom/android/server/LightsService;
  
      move-object/from16 v0, v75
  
!     invoke-direct {v0, v4}, Lcom/android/server/LightsService;-><init>(Landroid/content/Context;)V
      :try_end_6
      .catch Ljava/lang/RuntimeException; {:try_start_6 .. :try_end_6} :catch_1
  
--- 539,549 ----
  
      invoke-static {v3, v9}, Landroid/util/Slog;->i(Ljava/lang/String;Ljava/lang/String;)I
  
!     new-instance v75, Lcom/android/server/MiuiLightsService;
  
      move-object/from16 v0, v75
  
!     invoke-direct {v0, v4}, Lcom/android/server/MiuiLightsService;-><init>(Landroid/content/Context;)V
      :try_end_6
      .catch Ljava/lang/RuntimeException; {:try_start_6 .. :try_end_6} :catch_1
  
***************
*** 1375,1380 ****
--- 1378,1385 ----
      invoke-direct {v9, v4}, Lcom/android/server/DeviceStorageMonitorService;-><init>(Landroid/content/Context;)V
  
      invoke-static {v3, v9}, Landroid/os/ServiceManager;->addService(Ljava/lang/String;Landroid/os/IBinder;)V
+ 
+     invoke-static {}, Lcom/android/server/ServerThread$Injector;->setMemoryLowThresholdProperty()V
      :try_end_2f
      .catch Ljava/lang/Throwable; {:try_start_2f .. :try_end_2f} :catch_19
  
