*** ./smali/android/graphics/drawable/AnimationDrawable$AnimationState.smali	2013-10-21 03:28:00.511587237 +0800
--- /home/xiangxin/patchrom/MIUIV5/st25i/temp/new_smali/framework.jar.out/./smali/android/graphics/drawable/AnimationDrawable$AnimationState.smali	2013-10-21 03:29:43.123586679 +0800
***************
*** 1,15 ****
! .class final Landroid/graphics/drawable/AnimationDrawable$AnimationState;
  .super Landroid/graphics/drawable/DrawableContainer$DrawableContainerState;
  .source "AnimationDrawable.java"
  
  
  # annotations
  .annotation system Ldalvik/annotation/EnclosingClass;
      value = Landroid/graphics/drawable/AnimationDrawable;
  .end annotation
  
  .annotation system Ldalvik/annotation/InnerClass;
!     accessFlags = 0x1a
      name = "AnimationState"
  .end annotation
  
--- 1,19 ----
! .class public final Landroid/graphics/drawable/AnimationDrawable$AnimationState;
  .super Landroid/graphics/drawable/DrawableContainer$DrawableContainerState;
  .source "AnimationDrawable.java"
  
  
  # annotations
+ .annotation build Landroid/annotation/MiuiHook;
+     value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->CHANGE_ACCESS:Landroid/annotation/MiuiHook$MiuiHookType;
+ .end annotation
+ 
  .annotation system Ldalvik/annotation/EnclosingClass;
      value = Landroid/graphics/drawable/AnimationDrawable;
  .end annotation
  
  .annotation system Ldalvik/annotation/InnerClass;
!     accessFlags = 0x1c
      name = "AnimationState"
  .end annotation
  
***************
*** 171,173 ****
--- 175,193 ----
  
      return-object v0
  .end method
+ 
+ .method setDuration(II)V
+     .locals 1
+     .parameter "frame"
+     .parameter "value"
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->NEW_METHOD:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
+ 
+     .prologue
+     iget-object v0, p0, Landroid/graphics/drawable/AnimationDrawable$AnimationState;->mDurations:[I
+ 
+     aput p2, v0, p1
+ 
+     return-void
+ .end method
