*** ./smali/com/android/internal/telephony/IccPhoneBookInterfaceManagerProxy.smali	2013-10-21 03:28:53.959586947 +0800
--- /home/xiangxin/patchrom/MIUIV5/st25i/temp/new_smali/framework.jar.out/./smali/com/android/internal/telephony/IccPhoneBookInterfaceManagerProxy.smali	2013-10-21 03:30:42.307586358 +0800
***************
*** 35,40 ****
--- 35,53 ----
  
  
  # virtual methods
+ .method public getAdnCapacity()I
+     .locals 1
+ 
+     .prologue
+     iget-object v0, p0, Lcom/android/internal/telephony/IccPhoneBookInterfaceManagerProxy;->mIccPhoneBookInterfaceManager:Lcom/android/internal/telephony/IccPhoneBookInterfaceManager;
+ 
+     invoke-virtual {v0}, Lcom/android/internal/telephony/IccPhoneBookInterfaceManager;->getAdnCapacity()I
+ 
+     move-result v0
+ 
+     return v0
+ .end method
+ 
  .method public getAdnRecordsInEf(I)Ljava/util/List;
      .locals 1
      .parameter "efid"
***************
*** 83,88 ****
--- 96,114 ----
      return-object v0
  .end method
  
+ .method public getFreeAdn()I
+     .locals 1
+ 
+     .prologue
+     iget-object v0, p0, Lcom/android/internal/telephony/IccPhoneBookInterfaceManagerProxy;->mIccPhoneBookInterfaceManager:Lcom/android/internal/telephony/IccPhoneBookInterfaceManager;
+ 
+     invoke-virtual {v0}, Lcom/android/internal/telephony/IccPhoneBookInterfaceManager;->getFreeAdn()I
+ 
+     move-result v0
+ 
+     return v0
+ .end method
+ 
  .method public setmIccPhoneBookInterfaceManager(Lcom/android/internal/telephony/IccPhoneBookInterfaceManager;)V
      .locals 0
      .parameter "iccPhoneBookInterfaceManager"
