*** ./smali/android/view/ViewRootImpl$ViewRootHandler.smali	2013-10-21 03:28:20.619587128 +0800
--- /home/xiangxin/patchrom/MIUIV5/st25i/temp/new_smali/framework.jar.out/./smali/android/view/ViewRootImpl$ViewRootHandler.smali	2013-10-21 03:30:05.843586556 +0800
***************
*** 197,202 ****
--- 197,205 ----
  .method public handleMessage(Landroid/os/Message;)V
      .locals 13
      .parameter "msg"
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->CHANGE_CODE:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
  
      .prologue
      iget v1, p1, Landroid/os/Message;->what:I
***************
*** 307,312 ****
--- 310,323 ----
      .local v12, ri:Landroid/view/ViewRootImpl$ResizedInfo;
      iget-object v1, p0, Landroid/view/ViewRootImpl$ViewRootHandler;->this$0:Landroid/view/ViewRootImpl;
  
+     invoke-static {v1, v12, p1}, Landroid/view/ViewRootImpl$Injector;->tryToSkipResizedMsg(Landroid/view/ViewRootImpl;Landroid/view/ViewRootImpl$ResizedInfo;Landroid/os/Message;)Z
+ 
+     move-result v1
+ 
+     if-nez v1, :cond_0
+ 
+     iget-object v1, p0, Landroid/view/ViewRootImpl$ViewRootHandler;->this$0:Landroid/view/ViewRootImpl;
+ 
      iget-object v1, v1, Landroid/view/ViewRootImpl;->mWinFrame:Landroid/graphics/Rect;
  
      invoke-virtual {v1}, Landroid/graphics/Rect;->width()I
