*** ./smali/com/android/server/am/ActivityStack.smali	2013-10-21 03:27:17.355587472 +0800
--- /home/xiangxin/patchrom/MIUIV5/st25i/temp/new_smali/services.jar.out/./smali/com/android/server/am/ActivityStack.smali	2013-10-21 03:29:05.671586883 +0800
***************
*** 5432,5438 ****
      check-cast v3, Lcom/android/server/am/ActivityRecord;
  
      .local v3, r:Lcom/android/server/am/ActivityRecord;
!     iget-boolean v4, v3, Lcom/android/server/am/ActivityRecord;->finishing:Z
  
      if-eqz v4, :cond_1
  
--- 5432,5440 ----
      check-cast v3, Lcom/android/server/am/ActivityRecord;
  
      .local v3, r:Lcom/android/server/am/ActivityRecord;
!     invoke-static {p3, v3}, Lcom/android/server/am/ActivityStack$Injector;->isDestroyHomeReasonAlwaysOrFinishing(Ljava/lang/String;Lcom/android/server/am/ActivityRecord;)Z
! 
!     move-result v4
  
      if-eqz v4, :cond_1
  
***************
*** 6226,6231 ****
--- 6228,6236 ----
      .locals 7
      .parameter "r"
      .parameter "globalChanges"
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->CHANGE_CODE:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
  
      .prologue
      const/4 v3, 0x1
***************
*** 6316,6321 ****
--- 6321,6336 ----
      if-eqz v5, :cond_c
  
      :cond_7
+     iget-object v5, p1, Lcom/android/server/am/ActivityRecord;->info:Landroid/content/pm/ActivityInfo;
+ 
+     iget-object v5, v5, Landroid/content/pm/ActivityInfo;->packageName:Ljava/lang/String;
+ 
+     invoke-static {v5, v0, v1}, Landroid/app/MiuiThemeHelper;->needRestartActivity(Ljava/lang/String;ILandroid/content/res/Configuration;)Z
+ 
+     move-result v5
+ 
+     if-nez v5, :cond_0
+ 
      iget v5, p1, Lcom/android/server/am/ActivityRecord;->configChangeFlags:I
  
      or-int/2addr v5, v0
