*** ./smali/android/app/ContextImpl.smali	2013-10-21 03:28:14.443587162 +0800
--- /home/xiangxin/patchrom/MIUIV5/st25i/temp/new_smali/framework.jar.out/./smali/android/app/ContextImpl.smali	2013-10-21 03:29:59.291586592 +0800
***************
*** 8,14 ****
      value = {
          Landroid/app/ContextImpl$ApplicationContentResolver;,
          Landroid/app/ContextImpl$StaticServiceFetcher;,
!         Landroid/app/ContextImpl$ServiceFetcher;
      }
  .end annotation
  
--- 8,15 ----
      value = {
          Landroid/app/ContextImpl$ApplicationContentResolver;,
          Landroid/app/ContextImpl$StaticServiceFetcher;,
!         Landroid/app/ContextImpl$ServiceFetcher;,
!         Landroid/app/ContextImpl$Injector;
      }
  .end annotation
  
***************
*** 1089,1094 ****
--- 1090,1098 ----
      .parameter "broadcastPermission"
      .parameter "scheduler"
      .parameter "context"
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->CHANGE_CODE:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
  
      .prologue
      const/4 v7, 0x0
***************
*** 1137,1142 ****
--- 1141,1148 ----
      .restart local v3       #rd:Landroid/content/IIntentReceiver;
      :cond_1
      :goto_0
+     invoke-static {p0, p2}, Landroid/app/ContextImpl$Injector;->checkPriority(Landroid/app/ContextImpl;Landroid/content/IntentFilter;)V
+ 
      :try_start_0
      invoke-static {}, Landroid/app/ActivityManagerNative;->getDefault()Landroid/app/IActivityManager;
  
***************
*** 3475,3486 ****
  .end method
  
  .method final init(Landroid/app/LoadedApk;Landroid/os/IBinder;Landroid/app/ActivityThread;Landroid/content/res/Resources;Ljava/lang/String;)V
!     .locals 2
      .parameter "packageInfo"
      .parameter "activityToken"
      .parameter "mainThread"
      .parameter "container"
      .parameter "basePackageName"
  
      .prologue
      iput-object p1, p0, Landroid/app/ContextImpl;->mPackageInfo:Landroid/app/LoadedApk;
--- 3481,3495 ----
  .end method
  
  .method final init(Landroid/app/LoadedApk;Landroid/os/IBinder;Landroid/app/ActivityThread;Landroid/content/res/Resources;Ljava/lang/String;)V
!     .locals 3
      .parameter "packageInfo"
      .parameter "activityToken"
      .parameter "mainThread"
      .parameter "container"
      .parameter "basePackageName"
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->CHANGE_CODE:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
  
      .prologue
      iput-object p1, p0, Landroid/app/ContextImpl;->mPackageInfo:Landroid/app/LoadedApk;
***************
*** 3533,3539 ****
  
      move-result-object v1
  
!     invoke-virtual {p3, v0, v1}, Landroid/app/ActivityThread;->getTopLevelResources(Ljava/lang/String;Landroid/content/res/CompatibilityInfo;)Landroid/content/res/Resources;
  
      move-result-object v0
  
--- 3542,3552 ----
  
      move-result-object v1
  
!     iget-object v2, p0, Landroid/app/ContextImpl;->mPackageInfo:Landroid/app/LoadedApk;
! 
!     iget-object v2, v2, Landroid/app/LoadedApk;->mPackageName:Ljava/lang/String;
! 
!     invoke-virtual {p3, v2, v0, v1}, Landroid/app/ActivityThread;->getTopLevelResources(Ljava/lang/String;Ljava/lang/String;Landroid/content/res/CompatibilityInfo;)Landroid/content/res/Resources;
  
      move-result-object v0
  
