*** ./smali/android/app/INotificationManager.smali	2013-10-21 03:28:18.039587142 +0800
--- /home/xiangxin/patchrom/MIUIV5/st25i/temp/new_smali/framework.jar.out/./smali/android/app/INotificationManager.smali	2013-10-21 03:30:03.075586571 +0800
***************
*** 15,20 ****
--- 15,28 ----
  
  
  # virtual methods
+ .method public abstract areNotificationsEnabled(Ljava/lang/String;)Z
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
  .method public abstract areNotificationsEnabledForPackage(Ljava/lang/String;)Z
      .annotation system Ldalvik/annotation/Throws;
          value = {
