*** ./smali/android/widget/Toast$TN.smali	2013-10-21 03:27:39.631587351 +0800
--- /home/xiangxin/patchrom/MIUIV5/st25i/temp/new_smali/framework.jar.out/./smali/android/widget/Toast$TN.smali	2013-10-21 03:29:21.039586800 +0800
***************
*** 172,177 ****
--- 172,189 ----
  
  
  # virtual methods
+ .method getParams()Landroid/view/WindowManager$LayoutParams;
+     .locals 1
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->NEW_METHOD:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
+ 
+     .prologue
+     iget-object v0, p0, Landroid/widget/Toast$TN;->mParams:Landroid/view/WindowManager$LayoutParams;
+ 
+     return-object v0
+ .end method
+ 
  .method public handleHide()V
      .locals 2
  
