*** ./smali/com/android/internal/policy/impl/KeyguardUpdateMonitor$SimArgs.smali	2013-10-21 03:27:13.515587493 +0800
--- /home/xiangxin/patchrom/MIUIV5/st25i/temp/new_smali/android.policy.jar.out/./smali/com/android/internal/policy/impl/KeyguardUpdateMonitor$SimArgs.smali	2013-10-21 03:29:01.791586904 +0800
***************
*** 1,9 ****
! .class Lcom/android/internal/policy/impl/KeyguardUpdateMonitor$SimArgs;
  .super Ljava/lang/Object;
  .source "KeyguardUpdateMonitor.java"
  
  
  # annotations
  .annotation system Ldalvik/annotation/EnclosingClass;
      value = Lcom/android/internal/policy/impl/KeyguardUpdateMonitor;
  .end annotation
--- 1,13 ----
! .class public Lcom/android/internal/policy/impl/KeyguardUpdateMonitor$SimArgs;
  .super Ljava/lang/Object;
  .source "KeyguardUpdateMonitor.java"
  
  
  # annotations
+ .annotation build Landroid/annotation/MiuiHook;
+     value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->CHANGE_CODE:Landroid/annotation/MiuiHook$MiuiHookType;
+ .end annotation
+ 
  .annotation system Ldalvik/annotation/EnclosingClass;
      value = Lcom/android/internal/policy/impl/KeyguardUpdateMonitor;
  .end annotation
***************
*** 192,198 ****
  
      .end local v2           #state:Lcom/android/internal/telephony/IccCard$State;
      :cond_7
!     sget-object v2, Lcom/android/internal/telephony/IccCard$State;->UNKNOWN:Lcom/android/internal/telephony/IccCard$State;
  
      .restart local v2       #state:Lcom/android/internal/telephony/IccCard$State;
      goto :goto_0
--- 196,204 ----
  
      .end local v2           #state:Lcom/android/internal/telephony/IccCard$State;
      :cond_7
!     invoke-static {v3}, Lcom/android/internal/policy/impl/KeyguardUpdateMonitor$Injector;->getIccCardState(Ljava/lang/String;)Lcom/android/internal/telephony/IccCard$State;
! 
!     move-result-object v2
  
      .restart local v2       #state:Lcom/android/internal/telephony/IccCard$State;
      goto :goto_0
