*** ./smali/com/android/internal/view/menu/ActionMenuPresenter.smali	2013-10-21 03:28:52.055586957 +0800
--- /home/xiangxin/patchrom/MIUIV5/st25i/temp/new_smali/framework.jar.out/./smali/com/android/internal/view/menu/ActionMenuPresenter.smali	2013-10-21 03:30:39.987586370 +0800
***************
*** 15,21 ****
          Lcom/android/internal/view/menu/ActionMenuPresenter$ActionButtonSubmenu;,
          Lcom/android/internal/view/menu/ActionMenuPresenter$OverflowPopup;,
          Lcom/android/internal/view/menu/ActionMenuPresenter$OverflowMenuButton;,
!         Lcom/android/internal/view/menu/ActionMenuPresenter$SavedState;
      }
  .end annotation
  
--- 15,22 ----
          Lcom/android/internal/view/menu/ActionMenuPresenter$ActionButtonSubmenu;,
          Lcom/android/internal/view/menu/ActionMenuPresenter$OverflowPopup;,
          Lcom/android/internal/view/menu/ActionMenuPresenter$OverflowMenuButton;,
!         Lcom/android/internal/view/menu/ActionMenuPresenter$SavedState;,
!         Lcom/android/internal/view/menu/ActionMenuPresenter$Injector;
      }
  .end annotation
  
***************
*** 1047,1052 ****
--- 1048,1056 ----
      .locals 8
      .parameter "context"
      .parameter "menu"
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->CHANGE_CODE:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
  
      .prologue
      const/4 v7, 0x0
***************
*** 1069,1075 ****
  
      if-nez v4, :cond_0
  
!     invoke-virtual {v0}, Lcom/android/internal/view/ActionBarPolicy;->showsOverflowMenuButton()Z
  
      move-result v4
  
--- 1073,1079 ----
  
      if-nez v4, :cond_0
  
!     invoke-static {v0}, Lcom/android/internal/view/menu/ActionMenuPresenter$Injector;->showsOverflowMenuButton(Lcom/android/internal/view/ActionBarPolicy;)Z
  
      move-result v4
  
